docs/javascript/v2/get-started/javascript-quickstart.mdx

```diff
@@ -1091,9 +1124,7 @@ Refer to [Test the app section](/javascript/v2/guides/javascript-quickstart#test
 
 Here's the complete example.
 
-<Codesandbox id="5rmes" />
-
-There is also a version of the above quickstart using CDN [here](https://codepen.io/triptu/pen/dymxmoR?editors=0010).
+<Codesandbox id="github/100mslive/100ms-examples/tree/main/web/js-quickstart" />
```

Comment:
CDN link is removed?

Reply:
kept it back for now. Will update it in subsequent PRs. Right now keeping it for the sake for showing how to use our sdk using cdn link